src/mlpack/core/optimizers/sgd/test_function.cpp

```diff
@@ -35,6 +41,33 @@ double SGDTestFunction::Evaluate(const arma::mat& coordinates, const size_t i)
   }
 }
 
+double SGDTestFunction::Evaluate(const arma::mat& coordinates,
+                                 const size_t begin,
+                                 const size_t batchSize) const
+{
+  double objective = 0;
+
+  for (int i = begin; i < begin + batchSize; i++)
```

Comment:
Can you use ``size_t`` for the index parameter instead of ``int`` to be consistent with the rest of the codebase?